Docs/reference/content/reference/driver/ssl.md

```diff
@@ -54,8 +54,6 @@ default, setting [`CheckCertificateRevocation`]({{< apiref
 [here](https://docs.microsoft.com/en-us/dotnet/api/system.net.security.sslstream.authenticateasclient?view=netframework-4.7.2#System_Net_Security_SslStream_AuthenticateAsClient_System_String_)
 and .NET Standard documentation
 [here](https://docs.microsoft.com/en-us/dotnet/api/system.net.security.sslstream.authenticateasclient?view=netstandard-2.0#System_Net_Security_SslStream_AuthenticateAsClient_System_String_)).
-Applications relying on the intermediate default of `true` (introduced in beta driver releases 2.11.0-beta1 and 2.11.0-beta2) must
-explicitly set [`CheckCertificateRevocation`]({{< apiref
```

Comment:
I feel like you removed not enough. Looks like `"P_MongoDB_Driver_SslSettings_CheckCertificateRevocation">}})` should be removed as well. I need some time to build docs and check this.

Reply:
Yes you are right. I will look into this further...

Reply:
I think I should not have removed ANYTHING here....

Reply:
This text is impossible to read with all the embedded markup and hyperlinks... :)

Reply:
Yeah. I thought the point of removing this line is that no one really relies on beta release features. But it also wouldn't hurt to leave this line just in case.